genie-ddl/src/main/sql/mysql/schema.mysql.sql

```diff
@@ -120,16 +121,16 @@ CREATE TABLE `clusters` (
   `name` varchar(255) NOT NULL,
   `genie_user` varchar(255) NOT NULL,
   `version` varchar(255) NOT NULL,
-  `description` TEXT,
+  `description` text,
   `tags` varchar(10000) DEFAULT NULL,
   `setup_file` varchar(1024) DEFAULT NULL,
   `status` varchar(20) NOT NULL DEFAULT 'OUT_OF_SERVICE',
   `entity_version` int(11) DEFAULT '0',
   PRIMARY KEY (`id`),
   KEY `CLUSTERS_NAME_INDEX` (`name`),
-  KEY `CLUSTERS_TAG_INDEX` (`tags`),
-  KEY `CLUSTERS_STATUS_INDEX` (`status`)
-) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+  KEY `CLUSTERS_STATUS_INDEX` (`status`),
+  KEY `CLUSTERS_TAGS_INDEX` (`tags`(3072))
```

Comment:
This tags index specifies `KEY_BLOCK_SIZE`, unlike all other indices in this schema. Seems a bit strange, could this be left to default?

Reply:
I left it explicit here because it will flag when anyone running mysql 5.6 doesn't have the proper parameters setup. Which reminds me I need to add that to the documentation. In other words if they're running 5.6 with defaults and we say `INDEX(tags)` their index would only be 767 bytes long or whatever the default is without large_prefix_enabled set.